<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionsTab</class>
 <widget class="QWidget" name="SessionsTab">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTop">
     <item><widget class="QPushButton" name="buttonAddSession"><property name="text"><string>Ajouter</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonEditSession"><property name="text"><string>Modifier</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonDeleteSession"><property name="text"><string>Supprimer</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonExportSessions"><property name="text"><string>Exporter</string></property></widget></item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableSessions">
     <column>
      <property name="text"><string>Nom</string></property>
     </column>
     <column>
      <property name="text"><string>Début</string></property>
     </column>
     <column>
      <property name="text"><string>Fin</string></property>
     </column>
     <column>
      <property name="text"><string>Courante</string></property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSetCurrent">
     <property name="text"><string>Définir comme session courante</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>